src/install.rst

```diff
@@ -6,148 +6,184 @@ Installing Nextstrain
-      Next, try :doc:`tutorials/quickstart`.
+            nextstrain check-setup --set-default
 
-   .. group-tab:: Docker
+         The final output from the last command should look like this, where ``<option>`` is the environment chosen in the previous step:
 
-    `Docker <https://docker.com/>`_ is a container system freely-available for all platforms.
-    When you use the Nextstrain CLI with Docker, you don’t need to manage any other Nextstrain software dependencies as validated versions are already bundled into `a container image by the Nextstrain team <https://github.com/nextstrain/docker-base/>`_.
+         .. code-block:: none
 
-    First, `follow Docker's installation guide <https://docs.docker.com/engine/install/>`_ for your operating system.
-    After installing and starting Docker, create a Conda environment named ``nextstrain``.
-    This command will install the Nextstrain CLI and Git (a dependency of subsequent tutorials).
+            Setting default environment to <option>.
 
-    .. code-block:: bash
+   .. group-tab:: Windows
 
-        conda create -n nextstrain -c conda-forge -c bioconda nextstrain-cli git
+      .. note::
 
-    Confirm that the installation worked and configure the CLI to use Docker as the default environment manager.
+         Due to installation constraints, there is no way to use the native build/view environment on Windows directly. Follow steps for **WSL on Windows** if the native environment is desired.
 
-    .. code-block:: bash
+      1. Install `Anaconda or Miniconda <https://docs.conda.io/projects/conda/en/latest/user-guide/install/download.html>`_.
+         Miniconda is sufficient for this guide.
+      2. Install Visual C++ build tools following `this guide <https://stackoverflow.com/a/64262038>`_.
 
-        conda activate nextstrain
-        nextstrain check-setup --set-default
+         - This is necessary for a dependency of Nextstrain CLI and `we are investigating options to eliminate this step <https://github.com/nextstrain/cli/issues/31#issuecomment-970641263>`_.
 
-    The final output from the last command should look like this:
+      3. Open an Anaconda Prompt, which can be found in the Start menu.
+      4. Create a conda environment named ``nextstrain`` and install the Nextstrain CLI:
 
-    .. code-block:: bash
+         .. code-block:: none
 
-        Setting default environment to docker.
+            conda create -n nextstrain -c bioconda nextstrain-cli --yes
+            conda activate nextstrain
 
-    Finally, download the latest Docker image for Nextstrain.
+      5. Install the Nextstrain build/view environment by `installing Docker Desktop with WSL 2 backend <https://docs.docker.com/desktop/windows/wsl/>`_.
+      6. Confirm that the installation worked.
 
-    .. code-block:: bash
+         .. code-block:: none
 
-        nextstrain update
+            nextstrain check-setup --set-default
```

Comment:
When I tried this on a fresh Windows and Docker installation, this command failed while trying to pull the “hello-world” image with an error message about not having permission to access the image. Even though Docker Desktop was running, something was still broken. I opened the Docker dashboard and found a notice saying that I should login to have access to my images, etc. I logged in and ran the `check-setup` command again and it worked.

Did you have a similar experience, @victorlin, where you needed to login to Docker? Were you already logged in as a matter of habit?

Reply:
That's weird, I don't have a Docker account and never signed into Docker Desktop on macOS or Windows. Nevertheless, I'll look into setting up a new Windows user and going through these steps again for overall reproducibility to address other comments.

Reply:
I just re-checked and I haven't needed to login to Docker Desktop on any of the tested platforms. Let me know if you run into this again though, it could be beneficial to add in the troubleshooting section.